tests/fixtures/v4_hermes_plugin/__init__.py

```
@@ -0,0 +1,157 @@
+"""Tiny provider-free Hermes plugin used only by v4 gateway tests."""
+
+from __future__ import annotations
+
+import hashlib
+import json
+import os
+from pathlib import Path
+from typing import Any, Mapping
```

Ruff (0.16.3)

[warning] Import from `collections.abc` instead: `Mapping`

Import from `collections.abc`

(UP035)